pythainlp/corpus/common.py

```diff
@@ -336,3 +338,29 @@ def thai_synonyms() -> dict:
 def thai_synonym() -> dict:
     warnings.warn("Deprecated: Use thai_synonyms() instead.", DeprecationWarning)
     return thai_synonyms()
+
+
+def find_synonyms(word: str) -> List[str]:
+    """
+    Find synonyms
+
+    :param str word: Thai word
+    :return: List of synonyms of the input word or an empty list if it isn't exist.
+    :rtype: List[str]
+
+    :Example:
+    ::
+
+        from pythainlp.corpus import find_synonyms
+
+        print(find_synonyms("หมู"))
+        # output: ['จรุก', 'วราห์', 'วราหะ', 'ศูกร', 'สุกร']
+    """
+    synonyms = thai_synonyms()  # get a dictionary of {word, synonym}
+
+    if word in synonyms["word"]:
+        # returns the position of the first occurrence of the word
+        idx = synonyms["word"].index(word)
+        return synonyms["synonym"][idx]
```

Comment:
Approve.

Mind that currently it will only return the first occurrence set of synonyms. For example, from this synonyms list:
```python
{
    "word": ["สาว", "สาว"],
    "synonym": [["เด็ก"], ["ลาก", "ดึง"]]
}
```
`find_synonyms("สาว")` will return `["เด็ก"]`

Reply:
Oh no. My missing. It's still want to fix.

Reply:
OK. It's fixed.

```python
>>> from pythainlp.corpus import find_synonyms
>>> print(find_synonyms("ลูกสาว"))
['ดนยา', 'ธิดา', 'บุตรี', 'สุดา']
>>> print(find_synonyms("สุดา"))
['กระลาพิม', 'กระลาศรี', 'กัญญา', 'กันยา', 'กันย์', 'กัลยา', 'กัลยาณี', 'กานดา', 'กามินี', 'จอมขวัญ', 'จอมใจ', 'ฉัยยา', 'ชารี', 'ดนยา', 'ดรุณี', 'ดวงสมร', 'ถี', 'ทรามวัย', 'ทรามสงวน', 'ทรามสวาท', 'ทรามเชย', 'ธิดา', 'นงคราญ', 'นงพะงา', 'นงพาล', 'นงพุธ', 'นงราม', 'นงลักษณ์', 'นงเยาว์', 'นงโพธ', 'นรี', 'นารี', 'นาเรศ', 'นิรมล', 'บังอร', 'บุตรี', 'ผู้หญิง', 'พธู', 'พนิดา', 'พุ่มพวง', 'ภีรุ', 'มาณวิกา', 'มาตุคาม', 'มารศรี', 'ยุพดี', 'ยุพยง', 'ยุพา', 'ยุพาน', 'ยุพาพาล', 'ยุพาพิน', 'ยุพิน', 'ยุพเยาว์', 'ยุพเรศ', 'ยุวดี', 'รมณี', 'ร้อยชั่ง', 'ลลนา', 'ลูกสาว', 'วธู', 'วนิดา', 'วรดนู', 'สตรี', 'สมร', 'สะคราญ', 'สายสมร', 'สุดา', 'อนงค์', 'อร', 'อรทัย', 'อรนุช', 'อังคณา', 'อิตถี', 'อิสตรี', 'อิสัตรี', 'เยาวพา', 'เยาวมาลย์', 'เยาวลักษณ์', 'เยาวเรศ', 'แก้วตา', 'โฉมตรู', 'ไฉยา']
```